src/api/mock/data/article-collection.json

```diff
@@ -0,0 +1,48 @@
+{
+  "@context": "https://www.w3.org/ns/activitystreams",
+  "summary": "Sally's articles",
+  "type": "Collection",
```

Comment:
I think as we want to paginate I guess this would be the more appropriate type: https://www.w3.org/TR/activitystreams-vocabulary/#dfn-orderedcollection

Reply:
Isn't it better for paginate if we take CollectionPage or OrderedCollectionPage?

Reply:
Sorry completely missed out your comment. Yes, definitely OrderedCollectionPage will be the best to consume the inbox. See https://github.com/mitra-social/mitra-backend/pull/21/files#diff-7b12175e922fb64205a12ca86271e133R78-R86